Queries.sql

```diff
@@ -0,0 +1,40 @@
+-- Percentage of Conv2D layers per model
+
+SELECT Models.model_name, COUNT(operator_type) AS num_conv2d_layers
+FROM Models JOIN Operators
+ON Models.model_name = Operators.model_name
+WHERE operator_type = "Conv2D"
+GROUP BY model_name
+ORDER BY num_conv2d_layers DESC;
+
+-- Operator Distribution per problem category
```

Comment:
nit: These are for TFLite only? Perhaps add in comment?

Reply:
Added comment